### PR TITLE
Improve performance

### LINKED ISSUE #9: Remove console.log in the production code
There are a lot of them
https://github.com/ZimbiX/youtube-fast-fullscreen-toggle/blob/46ef67734cd53ecdc46c6035c5d812fc8b696da5/extension/contentScript.js#L30

### PR DESCRIPTION
**Pull request by aminya**
Fixes #9 

- Removes console log
- Uses built-in use-strict feature of the browser
- Fixes Eslint issues

### CODE REVIEW

File: extension/contentScript.js [hunk 1]
```diff
@@ -11,122 +11,108 @@
 // @grant        none
 // ==/UserScript==
 
-(function() {
```
Comment (ZimbiX) [inline extension/contentScript.js:14]: I'd thought enclosing the whole thing in a function was important to stop the scope leaking out, but I could be wrong there. Tampermonkey includes it in the template for a new script.

File: extension/contentScript.js [hunk 2]
```diff
@@ -11,122 +11,108 @@
...
 
-    const debugPlayerSize = (player) => {
-        console.log('[YT-FFT] w:', player.clientHeight, window.innerHeight, 'h:', player.clientWidth, window.innerWidth)
-    }
+const playerWidthDiff = (player) => Math.abs(player.clientWidth - window.innerWidth)
+const playerHeightDiff = (player) => Math.abs(player.clientHeight - window.innerHeight)
 
-    const playerIsFullscreen = () => {
-        const player = document.querySelector('ytd-player')
-        debugPlayerSize(player)
```
Comment (ZimbiX) [inline extension/contentScript.js:55]: Without any explanation as to how performance is improved, I assume you mean it's by not having to log the player size debugging? I never noticed an impact. And it'd be helpful information to supply when reporting an issue, so I'd like to leave it in. If you're concerned about it, a more useful change would be to make it toggleable via config.

File: extension/contentScript.js [hunk 3]
```diff
@@ -11,122 +11,108 @@
 // @grant        none
 // ==/UserScript==
 
-(function() {
-    'use strict';
+'use strict';
 
-    const isWatchVideoPage = () => window.location.pathname == "/watch"
+const isWatchVideoPage = () => window.location.pathname === "/watch"
```
Comment (ZimbiX) [inline extension/contentScript.js:16]: What's the necessity for all the triple equals?

**Review by ZimbiX — COMMENTED** [review]
Thanks for contributing, but I'm sorry, I don't think this is an improvement.

I don't like the formatting changes, and if anything, I'd be configuring eslint to prefer my style.

In future, please include more explanation and use smaller commits.

File: extension/contentScript.js (hunk 1 above)
Comment (aminya) [reply extension/contentScript.js:14]: Tampermonkey uses a very old template. In any browser used today, there is no need to use IEEE functions.

File: extension/contentScript.js (hunk 2 above)
Comment (aminya) [reply extension/contentScript.js:55]: `console.log` is useful when you are actively debugging the code. I don't think it is useful to print information and fill the dev console from these messages.

File: extension/contentScript.js (hunk 3 above)
Comment (aminya) [reply extension/contentScript.js:16]: === means that the type of the left-hand side is the same as the type of the right-hand side.

**Pull request closed without merging**